pkgs/development/python-modules/dogtag-pki/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchPypi, buildPythonPackage, pytest, python3, requests, six }:
```

```suggestion
{ stdenv, lib, fetchPypi, buildPythonPackage, pytest, requests, six }:
```
This potentially mixes python versions and is strictly forbidden.

---

gotcha.